sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Models/QuerySeriesRequestOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Azure.IoT.TimeSeriesInsights
+{
+    /// <summary>
+    /// Optional parameters to use when querying for Time Series Insights series.
+    /// </summary>
+    public class QuerySeriesRequestOptions : QueryRequestOptions
+    {
+        /// <summary>
+        /// Selected variables that needs to be projected in the query result. When it is null or not set, all the
+        /// variables and Time Series types in the model are returned.
+        /// </summary>
+        public string[] ProjectedVariables { get; set; }
```

Comment:
Should this be a list so items are easier to add/remove?

Reply:
Good idea. Will change.